Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "wormhole"
+version = "0.1.0"
+authors = ["user"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+quinn = { git = "https://github.com/zeroqn/quinn", branch = "expose-certificate" }
+thiserror = "1.0.9"
```

Comment:
Maby you should the ignore path version?  this's  always the style of the `muta-team`

Reply:
https://github.com/djc/quinn/issues/567
Haven't figure out a properly way to do this yet.